examples/example_FastMRI_Unet.py

```diff
@@ -0,0 +1,206 @@
+# %%
+"""
+Simple UNet model.
+
+This model is a simplified version of the U-Net architecture, 
+which is widely used for image segmentation tasks.  
+The U-Net model consists of an encoder (downsampling path) and 
+a decoder (upsampling path) with skip connections between corresponding 
+layers in the encoder and decoder. 
+These skip connections help in retaining spatial information 
+that is lost during the downsampling process.
+
+The primary purpose of this model is to perform image reconstruction tasks, 
+specifically for MRI images. 
+It takes an input MRI image and reconstructs it to improve the image quality 
+or to recover missing parts of the image.
+
+This implementation of the UNet model was pulled from the FastMRI Facebook 
+repository, which is a collaborative research project aimed at advancing 
+the field of medical imaging using machine learning techniques.
+
+References
+----------
+- [source Unet] FastMRI Facebook: https://github.com/facebookresearch/fastMRI/blob/main/fastmri/models/unet.py
+
+.. warning::
+    The training process is make on one image, so the training is not optimal.
```

Comment:
We train on a single image here. In practice, this should be done on a database like fastMRI [cite].